cmake/external/dml.cmake

```diff
@@ -95,6 +95,7 @@ else()
     add_dependencies(DirectML directml_repo-install)
     include_directories(BEFORE ${directml_install_path}/include)
   else()
-    include_directories(${dml_INCLUDE_DIR})
+    include_directories(BEFORE ${dml_INCLUDE_DIR})
+    set(DML_PACKAGE_DIR ${dml_INCLUDE_DIR}/..)
```

Comment:
(Correct if I miss anything)
We can only set this macro when we build DML from source because in all other cases binary has already been created, right?